docs/src/tut/community-insurance/index.md

```diff
@@ -0,0 +1,213 @@
+decides to terminate it. Initially this "contractIsRunning" variable is set to `true`. Later the deployer can decide to set it to `false`
+by interacting with the contract again through some API function. This API function will be defined and explained later in this tutorial.
++ Line 65, enables the insurer to know that the contract has been successflly deployed, at the time of deployment.
++ On Line 68, the insurer publishes the 2 values that were declassified in the local step (Lines 63, 64).
+An important note about the program states: On Line 67 we switched to concensus step because it is a "continuation" of `.only()` function. 
+Now on Line 68 we are calling the `.publish()` function which if called from a concensus step keeps us in the concensus step still. 
+Recall from the architecture that if this `publish()` function is called from a "step" then it changes the state of the program to "concensus-step".
+Therefore, we may conclude that this function always takes the program to a "concensus-step".
++ On Line 69, we are setting the invariant condition to `true` initially. 
+An invariant condition is a condition that evaluates to true before and after a while loop. We will need it later in our next piece of code.
+You can read [more about invariant conditions](https://docs.reach.sh/rsh/consensus/#term_loop%20invariant).
++ On Line 70 we are calling the `commit()` function to terminate/end the continuation of `only()` which was called earlier. 
+Remember, calling the `commit()` while in the concensus-step changes the program state to "step" mode. 
+The next steps in our program require us to be in a concensus-step mode, but we are now in "step" mode after calling `commit()`.
++ On Line 71 we are calling `Insurer.publish()` in order to switch back to a concensus-step.
+
++ Line 75 defines a [Set](https://docs.reach.sh/rsh/consensus/#rsh_Set) of items called `registeredMembers`, which will keep a linear list of all community members knonwn to this 
+insurance application. It only keeps their addresses, more info about the members is kept away from the blockchain (off-chain). 
++ Line 78 through 81 defines a [Map](https://docs.reach.sh/rsh/consensus/#rsh_Map) which will keep all open insurance claims. 
+Once a claim is funded or has expired, it will be deleted from the list. 
++ Lines 85 through 92 define a Map to keep the owners of the open claims (ie, commucity members). 
+
+Next, 
+```
+load: /examples/insurance/index.rsh
+range: range: 94-251
+```
+In this part of the program we define all the operations that a community member can execute as they declared previously in the 
+step part of the program where we specified the CommunityMember API.
+Here we are coding the logic of the CommunityMember API functions one by one. 
+Notice that we are using the [parallelReduce(...)](https://docs.reach.sh/rsh/consensus/#rsh_parallelReduce) function, which is one of the [concensus transfers](https://docs.reach.sh/guide/ctransfers/) in Reach. 
```

Comment:
Would suggest introducing as `parallelReduce`
Notice that we are using `parallelReduce`, which creates a race between `API` participants in a consensus step.